src/renderer/pages/player-page.js

```diff
@@ -227,6 +245,15 @@ function renderAudioMetadata (state) {
     ))
   }
 
+  // Audio metadata: disk & track-number
+  const count = ['track', 'disk']
+  count.forEach(key => {
+    const nrElem = renderTrack(common, key, key[0].toUpperCase() + key.substring(1))
```

Comment:
Maybe we can use CSS to capitalize the label.
If we do that we would only pass `key` and avoid operating directly on the string.

On `renderTrack` we would be able to do:
```
    const style = { textTransform: 'capitalize' }
    return (
      <div className={`audio-${key}`}>
        <label style={style}>{key}</label> {str}
      </div>
    )
```

Reply:
I have no experience with React, I blindly followed your suggestion.